index.html

```diff
@@ -23,56 +23,56 @@ <h3>Body</h3>
         <input type="text" name="" value="">
         <button type="button" name="button"></button>
         <div class="">
-          <button type="button" name="button"></button>
+          <button type="button" name="button"><img src="assets/search.svg" alt="Search Icon"></button>
           <input type="text" name="" value="">
         </div>
       </section>
       <section class="idea-cards">
         <div class="idea-box">
           <div class="box-header">
-            <img src="" alt="">
-            <img src="" alt="">
+            <img src="assets/star.svg" alt="Star Icon">
+            <img src="assets/delete.svg" alt="Delete Icon">
```

Comment:
This is just us adding the src for the images and alt codes, adding a little more to our HTML framework.